[2025-01-10T18:06:04.996+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-10T18:06:05.008+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stock_information.append_to_database scheduled__2025-01-10T18:05:00+00:00 [queued]>
[2025-01-10T18:06:05.015+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stock_information.append_to_database scheduled__2025-01-10T18:05:00+00:00 [queued]>
[2025-01-10T18:06:05.016+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-10T18:06:05.028+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): append_to_database> on 2025-01-10 18:05:00+00:00
[2025-01-10T18:06:05.034+0000] {standard_task_runner.py:72} INFO - Started process 2796 to run task
[2025-01-10T18:06:05.037+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'stock_information', 'append_to_database', 'scheduled__2025-01-10T18:05:00+00:00', '--job-id', '245', '--raw', '--subdir', 'DAGS_FOLDER/vnstock_stream.py', '--cfg-path', '/tmp/tmp81tazo4c']
[2025-01-10T18:06:05.039+0000] {standard_task_runner.py:105} INFO - Job 245: Subtask append_to_database
[2025-01-10T18:06:05.074+0000] {task_command.py:467} INFO - Running <TaskInstance: stock_information.append_to_database scheduled__2025-01-10T18:05:00+00:00 [running]> on host 453ac4defedc
[2025-01-10T18:06:05.151+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='TuanDz' AIRFLOW_CTX_DAG_ID='stock_information' AIRFLOW_CTX_TASK_ID='append_to_database' AIRFLOW_CTX_EXECUTION_DATE='2025-01-10T18:05:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-10T18:05:00+00:00'
[2025-01-10T18:06:05.154+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-10T18:06:05.204+0000] {logging_mixin.py:190} INFO - Kết nối thành công!
[2025-01-10T18:06:05.205+0000] {logging_mixin.py:190} INFO - Có lỗi khi thực hiện append_database: string indices must be integers, not 'list'
[2025-01-10T18:06:05.206+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-10T18:06:05.213+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-10T18:06:05.214+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=stock_information, task_id=append_to_database, run_id=scheduled__2025-01-10T18:05:00+00:00, execution_date=20250110T180500, start_date=20250110T180605, end_date=20250110T180605
[2025-01-10T18:06:05.251+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-10T18:06:05.263+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-10T18:06:05.266+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
